components/support/nimbus-fml/src/backends/kotlin/templates/FeatureManifestTemplate.kt

```diff
@@ -56,6 +56,11 @@ object {{ nimbus_object }} {
      */
     public fun initialize(getSdk: () -> FeaturesInterface?) {
         this.getSdk = getSdk
+
+        {%- for f in self.fm.iter_imported_files() %}
+        {{ f.about.nimbus_object_name_kt() }}.initialize(getSdk)
+        {%- endfor %}
+        this.invalidateCachedValues()
```

Comment:
just for my own clarity, why do we invalidate the cache here, when we expect consumers to invalidate it after the `applyPendingExperiments`?